backend/src/game/game.service.ts

```diff
@@ -43,133 +42,138 @@ export class GameService {
     this.startPaddle = canvasHeight / 2 - paddleHeight / 2;
   }
 
-  async joinOrCreateGame(
-    player: Player): Promise<[boolean, number]>{
+  async joinOrCreateGame(player: Player): Promise<[boolean, string]> {
     let pendingPlayer: string = await this.cacheManager.get('pendingPlayer');
-    
+
     // Check if pending player exists, is available and is not current player
-    if (pendingPlayer){
+    if (pendingPlayer) {
       const otherPlayer = JSON.parse(pendingPlayer);
       const user = await this.prisma.user.findUnique({
         where: {
           email: otherPlayer.email,
```

Comment:
I would change naming here. Maybe cachePendingPlayer and pendingPlayer - to make it more clear what is what.